web/src/engine/websites/ZeroScans.ts

```diff
@@ -56,8 +56,8 @@ export default class extends DecoratableMangaScraper {
 
     public override async FetchManga(provider: MangaPlugin, url: string): Promise<Manga> {
         const request = new FetchRequest(url);
-        const data = await FetchWindowScript<APIMangaClipboard>(request, `__ZEROSCANS__`);
-        return new Manga(this, provider, JSON.stringify({ id: data.data[0].details.id, slug: data.data[0].details.slug }), data.data[0].details.name.trim());
+        const { data } = await FetchWindowScript<APIMangaClipboard>(request, `__ZEROSCANS__`, 2000);
```

Comment:
Suggestion to decompose to `details` since data seems not to be used
```typescript
const { _data: [ details ] } = ... // details.id, details.slug, details.name
```